device/arista/x86_64-arista_7060x6_16pe_384c/platform.json

```diff
@@ -690,7 +690,7 @@
             "lanes": "457,458,459,460",
```

Comment:
if we need to update the port number, should we update the index here as well?

I think this is what Prince G was saying.